docs/source/configuration/instruments.mdx

```diff
@@ -0,0 +1,318 @@
+        value:
+         request_header: "x-my-header"
+```
+
+<Note>
+
+The value must be of the expected type for the instrument. For example, a counter must have a numeric value.
+
+</Note>
+
+#### `type`
+
+Instruments come in four different types:
+
+* `counter` - A monotonic counter. For example, requests served, tasks completed, or errors occurred.
+* `up_down_counter` - A bidirectional counter that is modified by a delta.  For example, the number of items in a queue.   
+* `gauge` - A value that is sampled at a specific point in time. For example, active requests or CPU usage. 
+* `histogram` - A histogram of values. For example, request durations or response body sizes.
+
+```yaml title="router.yaml"
+telemetry:
+  instruments:
+    router:
+      acme.metric: 
+        # ...
+        type: counter # counter, up_down_counter, gauge, histogram
+```
+
+#### `unit`
+
+A free format unit that is displayed in your APM.
+
+A `unit` is recommended to use SI units and definitions from [The Unified Code for Units of Measure](https://ucum.org/ucum).
+
+```yaml title="router.yaml"
+telemetry:
+  instruments:
+    router:
+      acme.metric: 
+        # ...
+        unit: s # seconds
+```
+
+#### `description`
+
+A free format description of the instrument that will be displayed in your APM.
+
+```yaml title="router.yaml"
+telemetry:
+  instruments:
+    router:
+      acme.metric: 
+        # ...
+        description: "my description"
+```
+
+#### `condition`
+
+You may only want to mutate an instrument under certain conditions. For example, you may only want to increment a counter if the response status code is 200.
+
+To do this use a condition:
+
+```yaml title="router.yaml"
+telemetry:
+  instruments:
+    router:
+      acme.metric:
+        # ...
+        condition:
+          eq:
+            - 200
+            - response_header: Status
+```
+
+#### `attributes`
+
+Instruments may have attributes attached to them from the router pipeline. These attributes are used to filter and group metrics in your APM. 
+
+Attributes may be drawn from [standard attributes](./standard-attributes) or [selectors](./selectors).
+
+The attributes available depend on the service of the pipeline.
+
+```yaml title="router.yaml"
+telemetry:
+  instruments:
+    router:
+      # Standard metrics
+      http.server.active_requests:
+        attributes:
+          # Standard attributes
+          http.response.status_code: false
+          # Custom attribute
+          "acme.my_attribute":
+            response_header: "x-my-header"
+      # Custom metric 
+      acme.metric: 
+        attributes:
+          http.response.status_code: true
+          "my_attribute":
+            # ...
+            response_header: "x-my-header"
+```
+
+### Instrument configuration reference
+
+| Option                                | Values                                                                       | Default    | Description                                  |
```

Comment:
This table currently needs to be horizontal scrolled to view the descriptions, we can probably find a better format.